script/test.sh

```diff
@@ -164,6 +164,29 @@ test_ls() {
   teardown
 }
 
+# https://github.com/kamontia/qs/issues/71
+test_range_validation() {
+  FLAGS="$1 $2"
+  RANGE="$3"
+  MESSAGE="$4" # Expect to finish leaving this message(test target)
+  VALIDATION_FILE="test_range_validation.dat" # Temporary file to validate this test.
+
+  setup
+
+  set +e
+  ./"$EXEC_COMMAND" ${FLAGS} ${RANGE} | tee ${VALIDATION_FILE}
```

Comment:
it is better to user mktemp.
but no problem because this script runs in series.